tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-product-list-table-test.php

```
@@ -0,0 +1,491 @@
+	/**
+	 * Scenarios for {@see self::test_save_quick_edit()}.
+	 */
+	public function provide_quick_edit_save_scenarios(): array {
+		return [
+			'checked checkbox persists yes'         => [ 'simple', '', 'yes', true, 'yes' ],
+			'absent checkbox collapses to no'       => [ 'simple', 'yes', null, true, 'no' ],
+			'tampered value collapses to no'        => [ 'simple', 'yes', 'forged', true, 'no' ],
+			'unsupported type is left untouched'    => [ 'grouped', 'yes', null, true, 'yes' ],
+			'feature off leaves stored value alone' => [ 'simple', 'yes', null, false, 'yes' ],
+		];
+	}
```

PHPMD (2.15.0)

[error] The method provide_quick_edit_save_scenarios is not named in camelCase. (undefined)

(CamelCaseMethodName)